agent/consul/leader.go

```diff
@@ -1318,6 +1318,16 @@ func (s *Server) handleAliveMember(member serf.Member) error {
 			ID:      structs.ConsulServiceID,
 			Service: structs.ConsulServiceName,
 			Port:    parts.Port,
+			Weights: &structs.Weights{
+				Passing: 1,
+				Warning: 1,
+			},
+			Meta: map[string]string{
+				"consul_voter":  strconv.FormatBool(!parts.NonVoter),
```

Comment:
This can change at runtime and I wonder if it should be included here or maybe made more explicit by changing the key to something like `booted_as_consul_voter`.